chalicelib/services/BroadcastService.py

```diff
@@ -0,0 +1,84 @@
+from chalicelib.modules.ses import ses, SesDestination
+from chalicelib.services.JobPostingService import job_posting_service
+from typing import List, Dict
+
+class BroadcastService:
+    def __init__(self):
+        self.ps = job_posting_service
+    
+    def generate_job_html(self, jobs: List[Dict]) -> str:
+        """Generates HTML for job listings based on the new job data format."""
+        html = "<div class='jobs-section'>"
+        html += "<h2>Job Opportunities</h2>"
+        html += "<ul class='job-list'>"
+        
+        for job in jobs:
+            company = job.get("company", "N/A")
+            role = job.get("role", "N/A")
+            link = job.get("link", "#")
+            deadline = job.get("date", "N/A")
+            
+            job_line = (
```

Comment:
nit: What do you think about wrapping the entire line into a text instead just to save vertical space? @wderocco8